src/common/src/types/pipeline_tags.rs

```diff
@@ -34,9 +34,9 @@ pub struct PipelineTags {
 impl Default for PipelineTags {
     fn default() -> Self {
         Self {
-            environment: "local".into(),
-            pipeline_type: "generic".into(),
-            user_operator: "tracer_user".into(),
+            environment: Some("local".into()),
+            pipeline_type: Some("generic".into()),
+            user_operator: Some(whoami::username()),
```

Comment:
<img width="573" alt="image" src="https://github.com/user-attachments/assets/f2e5c7a0-58e4-4de3-990a-d9e4e3add922" />

in macos there is the $USER env variable, maybe for linux is the same as well, and we can avoid using other dependency for this simple stuff. 

But not really urgent